packages/sui-ssr/server/index.js

```diff
@@ -119,8 +121,8 @@ const _memoizedHtmlTemplatesMapping = {}
   app.use(hooks[TYPES.PRE_SSR_HANDLER])
 
   app.get('*', [
-    criticalCss(ssrConf.criticalCSS),
-    prpl(ssrConf.prpl),
+    // criticalCss(ssrConf.criticalCSS),
+    // prpl(ssrConf.prpl),
```

Comment:
Don't comment this, because you're are disabling those features in other sites. Better just remove the config from our package.json